kview/src/main/java/dev/ikm/komet/kview/lidr/mvvm/model/DataModelHelper.java

```diff
@@ -92,13 +96,36 @@ public class DataModelHelper {
             Entity.getFast(TinkarTerm.DESCRIPTION_INITIAL_CHARACTER_CASE_SENSITIVE.nid())
     );
 
+    public static final Set<ConceptEntity> DATA_TYPE_OPTIONS = Set.of(
```

Comment:
can we make a DataModelHelper.java in the mvvm.model package so that it doesn't live in the in the lidr.mvvm.model?

Reply:
> this will add additional dependencies and configuration for a small temp fix.

Reply:
the idea is that lidr is supposed to be completely separate from the other mvvm stuff

Reply:
This is correct. LIDR will be a separate plugin.

Reply:
Done